.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+# Dependabot configuration.
+#
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Let's set a time (default day is Monday, default timezone is UTC), so that we know when the PRs should have arrived if they were coming for that week.

```suggestion
      interval: "weekly"
      time: "09:00"
```

---

I've added it, but I previously didn't add it on purpose. If everyone sets a time, especially at a whole hour, this causes a peak burden on the GH servers, which may have unintended side-effects. Leaving the time open, you still know that the PRs should have arrived by midnight UTC on Monday, but it allows GH/Dependabot to sort out their queue with more flexibility.